From 9df3ebb9aa06a476cc4b88f7178b5651e2460895 Mon Sep 17 00:00:00 2001
From: salahkarout <s_karout@hotmail.co.uk>
Date: Mon, 11 Jul 2016 11:17:01 +0100
Subject: [PATCH] Added my_strcat function

---
 src/string_operations.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/string_operations.c b/src/string_operations.c
index 68f7abc..ba9df81 100644
--- a/src/string_operations.c
+++ b/src/string_operations.c
@@ -22,6 +22,7 @@ char *my_strcpy(char *t, char *s)
    
 }
 
+
 int main(void)
 {
 	int i;
@@ -40,4 +41,4 @@ int main(void)
         printf("%s\n", my_strcpy(p1, "Hello, World !!!"));
 
 	return 0;
-}
\ No newline at end of file
+}
-- 
2.7.2.windows.1

